master/cli-reference/ark_server.md~ba18ad8... add v0.9.0 docs to site

```diff
@@ -0,0 +1,40 @@
+## ark server
```

Comment:
This looks like a garbage file (git leftover from something or other), should be removed